src/PythonBindings/VectorPyList.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <vector>
+
+#include <boost/python.hpp>
+#include <boost/python/stl_iterator.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+
+namespace bp = boost::python;
+/*!
```

Comment:
We should make sure we only document this namespace once. Here is probably not the right place.

Reply:
How should this be documented?  Should it just not be in this pull request?